src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -80,9 +80,18 @@ private static Request CreateRequest(FileConfig fileConfig, string absoluteFileP
             Debug.Assert(request.RulesConfiguration != null, "RulesConfiguration should be set for the analysis request");
             request.Options = GetKeyValueOptionsList(request.RulesConfiguration);
 
-            if (analyzerOptions is CFamilyAnalyzerOptions cFamilyAnalyzerOptions && cFamilyAnalyzerOptions.CreateReproducer)
+            if (analyzerOptions is CFamilyAnalyzerOptions cFamilyAnalyzerOptions)
             {
-                request.Flags |= Request.CreateReproducer;
+                if (cFamilyAnalyzerOptions.CreateReproducer)
+                {
+                    request.Flags |= Request.CreateReproducer;
+                }
+
+                if (cFamilyAnalyzerOptions.CreatePreCompiledHeaders)
+                {
+                    request.Flags |= Request.BuildPreamble;
+                    request.PchFile = cFamilyAnalyzerOptions.PreCompiledHeadersFilePath;
```

Comment:
It's worth adding at least some assertions to validate the arguments i.e. that the `CreateReproducer` and `CreatePreCompilerHeaders` are not both passed, and that the path isn't null.